src/style.css

```diff
@@ -0,0 +1,75 @@
+body {
+  font-weight: 300;
+}
+
+/* @font-face {
+  font-family: Avenir;
+  src: url();
+} */
+
+::placeholder {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.big-search-image {
+  background-image: url(./images/Search.svg);
+  height: 19px;
+  width: 19px;
+  position: absolute;
+  left: 34px;
+  top: 45px;
+}
+
+.small-search-image {
+  background-image: url(./images/Search.svg);
+  height: 11px;
+  width: 11px;
+  position: absolute;
+  left: 23px;
+  top: 125px;
+  background-size: cover;
+}
```

Comment:
You repeated properties background-image and position. It is bad practice. Move it to common class and use this class